include/z64save.h

```diff
@@ -888,28 +888,32 @@ typedef enum {
 // Cremia did Milk Run alone. Player didn't interact or didn't accept the ride
 #define WEEKEVENTREG_34_80 PACK_WEEKEVENTREG_FLAG(34, 0x80)
 
-#define WEEKEVENTREG_35_01 PACK_WEEKEVENTREG_FLAG(35, 0x01)
-#define WEEKEVENTREG_35_02 PACK_WEEKEVENTREG_FLAG(35, 0x02)
-#define WEEKEVENTREG_35_04 PACK_WEEKEVENTREG_FLAG(35, 0x04)
-#define WEEKEVENTREG_35_08 PACK_WEEKEVENTREG_FLAG(35, 0x08)
-#define WEEKEVENTREG_35_10 PACK_WEEKEVENTREG_FLAG(35, 0x10)
-#define WEEKEVENTREG_35_20 PACK_WEEKEVENTREG_FLAG(35, 0x20)
+// Bought each possible map from Tingle
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_CLOCK_TOWN PACK_WEEKEVENTREG_FLAG(35, 0x01)
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_WOODFALL PACK_WEEKEVENTREG_FLAG(35, 0x02)
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_SNOWHEAD PACK_WEEKEVENTREG_FLAG(35, 0x04)
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_ROMANI_RANCH PACK_WEEKEVENTREG_FLAG(35, 0x08)
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_GREAT_BAY PACK_WEEKEVENTREG_FLAG(35, 0x10)
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_STONE_TOWER PACK_WEEKEVENTREG_FLAG(35, 0x20)
+
 #define WEEKEVENTREG_35_40 PACK_WEEKEVENTREG_FLAG(35, 0x40)
 
 // Obtained Heart Piece from Five Frogs of the Frog Choir
 #define WEEKEVENTREG_35_80 PACK_WEEKEVENTREG_FLAG(35, 0x80)
 
-#define WEEKEVENTREG_36_01 PACK_WEEKEVENTREG_FLAG(36, 0x01)
-#define WEEKEVENTREG_36_02 PACK_WEEKEVENTREG_FLAG(36, 0x02)
-#define WEEKEVENTREG_36_04 PACK_WEEKEVENTREG_FLAG(36, 0x04)
-#define WEEKEVENTREG_36_08 PACK_WEEKEVENTREG_FLAG(36, 0x08)
-#define WEEKEVENTREG_36_10 PACK_WEEKEVENTREG_FLAG(36, 0x10)
-#define WEEKEVENTREG_36_20 PACK_WEEKEVENTREG_FLAG(36, 0x20)
-#define WEEKEVENTREG_36_40 PACK_WEEKEVENTREG_FLAG(36, 0x40)
-#define WEEKEVENTREG_36_80 PACK_WEEKEVENTREG_FLAG(36, 0x80)
-#define WEEKEVENTREG_37_01 PACK_WEEKEVENTREG_FLAG(37, 0x01)
-#define WEEKEVENTREG_37_02 PACK_WEEKEVENTREG_FLAG(37, 0x02)
-#define WEEKEVENTREG_37_04 PACK_WEEKEVENTREG_FLAG(37, 0x04)
+// Player has spoken to certain shrine gorons in the winter
+#define WEEKEVENTREG_TALKED_TO_WINTER_SHRINE_GORON_A_AS_NONGORON_SON_CRYING PACK_WEEKEVENTREG_FLAG(36, 0x01)
+#define WEEKEVENTREG_TALKED_TO_WINTER_SHRINE_GORON_A_AS_GORON_SON_CRYING PACK_WEEKEVENTREG_FLAG(36, 0x02)
```

Comment:
Ah, I can see where this comes from since waffleoRai decompiled EnGoro Previously? But yeah, might be better in a different PR. (Having a separate PR just for these flags is totally fine!)

Reply:
Ah sorry! Yes, it was because I did EnSGoro previously and usually leave the .c open in VS for reference. Figured I'd add the other flags that I knew about while I was at it. I'll move those changes to a different branch.